requirements.txt

```diff
@@ -1 +1,2 @@
-python-dateutil==2.6.0
\ No newline at end of file
+python-dateutil==2.6.0
+inflection
```

Comment:
I think this should get a version tag, just in case of future bugs or licensing changes.

Reply:
this is also a [single file](https://github.com/jpvanhal/inflection/blob/master/inflection.py), would it make sense instead to bundle it in lingua franca? the advantage would be for other languages to use as template

otherwise i agree that we should pin version number